### ISSUE TITLE
"New" icon's position

### ISSUE THREAD

**Issue by Hristo-** [SimpleMachines/SMF#1039]
IIRC the new icon in 2.1 used to be placed before the subject, like this:
![new_before](https://f.cloud.github.com/assets/6107569/1743067/817ef81c-6406-11e3-91d0-ae174bfa5172.png)

At the moment it's back again as it is in 2.0.*:
![new_now](https://f.cloud.github.com/assets/6107569/1743075/a2abe784-6406-11e3-9af3-9bd127e32387.png)

I remember I liked a lot it' s position in front, because that way it was easier to spot which topics are not read. Because subjects are of different length it's kind of hard to follow which topics are not read. That is especially true for the default 2.0.\* message index, because it lacks visual borders between rows. 2.1 is better in that regard, but still IMO the in front position is better. What are your thoughts?

Edit: That was stupid by me, 2.0.\* message index has borders between rows, lighter, but still has. That only confirms (in my eyes) that 2.1 will benefit none the less if the icon is in front.

**Comment by Arantor** [comment]
I also tend to think that it will help with usability on a touch screen by giving you an area to hit with less chance of other links being in the way (certainly that's been my experience of using this site on an iPad)

**Comment by Xarcell** [comment]
I agree that it should go in the front.
